pkg/cmd/login/login.go

```diff
@@ -57,7 +57,7 @@ func NewCmdLogin(f factory.Factory) *cobra.Command {
 		Long:  "Login to your Octopus server using OpenID Connect (OIDC) or an API key. If no arguments are provided then login will be done interactively allowing creation of an API key.",
 		Example: heredoc.Docf(`
 			$ %[1]s login
-			$ %[1]s login --server https://my.octopus.app --service-account-id b1a6f20f-0ec7-4e9a-938e-db800f945b37 --id-token eyJhbGciOiJQUzI1NiIsImtp...
```

Comment:
Imtp was coming up as a misspelt word

Reply:
and `NiIs` isn't 🤯 🤷

Reply:
The captial `I` starts a new word

Reply:
![](https://media.giphy.com/media/ALBfFB6gP1evu/giphy.gif)